src/routes.js

```diff
@@ -152,5 +152,9 @@ export const communityOnboardingUrl = community =>
 export const commentUrl = comment =>
   `/p/${comment.post_id}#comment-${comment.id}`
 
-export const tagUrl = (name, slug) =>
-  slug ? `/c/${slug}/tag/${name}` : `/tag/${name}`
+export const tagUrl = (name, slug) => {
```

Comment:
I thought about that, but it's getting a little unreadable for me. I can change it, your call...